HCL.IdentityServer.API.Test/IntegrationTest/Services/AccountServiceIntegrationTest.cs

```diff
@@ -0,0 +1,315 @@
+﻿using DotNet.Testcontainers.Containers;
+using FluentAssertions;
+using HCL.IdentityServer.API.BLL.Services;
+using HCL.IdentityServer.API.DAL;
+using HCL.IdentityServer.API.DAL.Repositories;
+using HCL.IdentityServer.API.Domain.Entities;
+using HCL.IdentityServer.API.Domain.Enums;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace HCL.IdentityServer.API.Test.IntegrationTest.Services
+{
+    public class AccountServiceIntegrationTest : IAsyncLifetime
+    {
+        IContainer pgContainer = TestContainerBuilder.CreatePostgreSQLContainer();
+        private WebApplicationFactory<Program> webHost;
+
+        public async Task InitializeAsync()
+        {
+            await pgContainer.StartAsync();
+            webHost = CustomTestHostBuilder.BuildWithAdmin(TestContainerBuilder.npgsqlUser, TestContainerBuilder.npgsqlPassword
+                , "localhost", pgContainer.GetMappedPublicPort(5432), TestContainerBuilder.npgsqlDB);
+        }
+
+        public async Task DisposeAsync()
+        {
+            await pgContainer.StopAsync();
+        }
+
+        [Fact]
+        public async Task AddAccount_WithRightData_ReturnNewAccount()
+        {
+            //Arrange
+            using var scope = webHost.Services.CreateScope();
+            var appDBContext = scope.ServiceProvider.GetRequiredService<AppDBContext>();
+            var accRep = new AccountRepository(appDBContext);
```

Comment:
it is better to write full names of variables. (accRep - accountRepository, accountServ - accountService)